### ISSUE TITLE
Lazy Loading Images 

### ISSUE THREAD

**Issue by prashantsani** [IanLunn/Sequence#256]
Hello.
I m using this slider for a website that has nearly 30 slides.

Can we have lazy-loading built in inside slider?
I m also using hash in options, 

          `hashTags:true,
          hashDataAttribute:true,
          hashChangesOnFirstFrame:true,`

So, when the user goes **somepage.html#slide20** , the slider will show pre-loader and load images inside slide20.

**Comment by IanLunn** [comment]
Yes, will think about lazy loading in the future for v2. You can probably put something together using the API and `preloadTheseFrames` [option](http://www.sequencejs.com/developers/documentation/#preloader-options).
